x-pack/plugins/secops/server/lib/network/query_dns.dsl.ts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Direction, NetworkDnsFields, NetworkDnsSortField } from '../../graphql/types';
+import { createQueryFilterClauses } from '../../utils/build_query';
+
+import { NetworkDnsRequestOptions } from './index';
+
+type DslSort = 'asc' | 'desc';
```

Comment:
Can we use the type `SortRequestDirection` here?

Reply:
we get refactor in next PR just after this one